app/controllers/api/api.js

```diff
@@ -18,4 +19,66 @@ function authenticatedUser(req, res, next) {
 
 router.get('/secret', authenticatedUser, function (req, res, next) {
   res.json({message: "secret"});
+});
+
+//INDEX
+router.get('/api/bottles', authenticatedUser,function(req, res){
+  Bottle.find({user_id: req.user.id}, function(err, bottles){
+    if (err){
+      res.send("Something Wrong happened" + err)
+    } else {
+      res.send(bottles);
+    }
+  });
+})
+
+//CREATE
+router.post('/api/bottles', authenticatedUser, function (req, res){
+  var bottleParams = req.body.bottle;
+  bottleParams.user_id = req.user.id;
+  Bottle.create(bottleParams, function (err, bottles){
+    if (err) {
+      res.send("something wrong happened " + err )
+    } else {
+      //update user model's bottle id
+      // Bottle.findby({user_id: req.user._id}, function (err, bottle){
+      //   req.body.user.bottle_id = bottle.id;
+      // })
+      res.redirect('/api/bottles');
```

Comment:
api shouldn't have redirects. it should only have .json... so please enter a message saying it is created OR return the `bottle` object